docs/specs/clients/sign/client-api.md

```diff
@@ -74,6 +74,33 @@ abstract class Client {
     reason: Reason;
   }): Promise<void>;
 
+  // ---------- Wallet Authentication Methods ----------------------------------------------- //
+
+  // for proposer to request wallet authentication
+  public abstract authenticate(params: {
```

Comment:
Will ReCaps be added on the next PR/step?

Reply:
Yes, this is the first step, after ReCaps standard & specs